compiler/rustc_feature/src/unstable.rs

```diff
@@ -623,6 +623,8 @@ declare_features! (
     /// Allows creation of instances of a struct by moving fields that have
     /// not changed from prior instances of the same struct (RFC #2528)
     (unstable, type_changing_struct_update, "1.58.0", Some(86555)),
+    /// Allows declaring fields `unsafe`.
+    (incomplete, unsafe_fields, "CURRENT_RUSTC_VERSION", Some(1234567)), // FIXME: proper tracking issue.
```

Comment:
We have a tracking issue now!
```suggestion
    (incomplete, unsafe_fields, "CURRENT_RUSTC_VERSION", Some(132922)),
```